src/messages/en.json

```
@@ -75,10 +75,11 @@
     }
   },
   "ReportChapterPage": {
-    "back": "Back"
+    "back": "Back",
+    "tocTitle": "On This Page"
```

```suggestion
    "tocTitle": "On This Chapter"
```
we are talking about the chapter detail page, aren't we? :P